src/argilla/client/client.py

```diff
@@ -490,6 +492,13 @@ def load(
                 can be used to load using batches.
             as_pandas: DEPRECATED! To get a pandas DataFrame do
                 ``rg.load('my_dataset').to_pandas()``.
+            batch_size: If provided, load `batch_size` samples per request. A lower batch
+                size may help avoid timeouts.
+            exclude_vectors: When set to `True`, indicates that the record data will be retrieved excluding its vectors,
+                if any. By default, this parameter is set to `False`, meaning that vector data will be included.
+            exclude_metrics: When set to `True`, indicates that the record data will be retrieved excluding its metrics.
+                By default, this parameter is set to `False`, meaning that metrics will be included.
```

Comment:
```suggestion
            batch_size: If provided, load `batch_size` samples per request. A lower batch
                size may help avoid timeouts.
            exclude_vectors: When set to `True`, indicates that the record data will be retrieved excluding its vectors,
                if any. By default, this parameter is set to `False`, meaning that vector data will be included.
            exclude_metrics: When set to `True`, indicates that the record data will be retrieved excluding its metrics.
                By default, this parameter is set to `False`, meaning that metrics will be included.
            as_pandas: DEPRECATED! To get a pandas DataFrame do
                ``rg.load('my_dataset').to_pandas()``.
```
Let's stick with the ordering from the method signature.
Edit: Wow, the diff looks like a mess. I've just moved `as_pandas` down to below `exclude_metrics`.